src/simulation/Game.ts

```diff
@@ -111,6 +120,10 @@ export default class Game extends GameLoop {
 
   populateCell(x: number, y: number): GameCell {
     const gameCell = new GameCell(this.world.getCell(x, y));
+    gameCell.newPop$.subscribe((pop) => {
```

Comment:
Instead of creating a new subscription every time you call this function, it would be better if we used RXJS operators here, like [mergeMap](https://www.learnrxjs.io/operators/transformation/mergemap.html) maybe.